concepticondata/conceptlists/Marchese-1983-311.tsv

```diff
@@ -0,0 +1,312 @@
+Marchese-1983-311-243	243	chanter	1261	SING	to sing	LES VERBES
+Marchese-1983-311-244	244	chercher	1468	LOOK FOR	to search	LES VERBES
+Marchese-1983-311-245	245	choisir	1750	CHOOSE	to choose	LES VERBES
+Marchese-1983-311-246	246	comprendre	1536	UNDERSTAND	to understand	LES VERBES
+Marchese-1983-311-247	247	compter	1420	COUNT	to count	LES VERBES
+Marchese-1983-311-248	248	coucher	2071	LAY (VERB)	to lay down	LES VERBES
+Marchese-1983-311-249	249	couler	2003	FLOW	to flow	LES VERBES
+Marchese-1983-311-250	250	couper	1432	CUT	to cut	LES VERBES
+Marchese-1983-311-251	251	creuser	1418	DIG	to dig	LES VERBES
+Marchese-1983-311-252	252	crier	715	SHOUT	to yell	LES VERBES
+Marchese-1983-311-253	253	cuire (préparer	1100	COOK (SOMETHING)	to cook (prepare	LES VERBES
+Marchese-1983-311-254	254	être debout	1442	STAND	to stand	LES VERBES
+Marchese-1983-311-255	255	déféquer	676	SHIT (DEFECATE)	to defecate	LES VERBES
+Marchese-1983-311-256	256	descendre	2014	GO DOWN (DESCEND)	to go down	LES VERBES
+Marchese-1983-311-257	257	donner	1447	GIVE	to give	LES VERBES
+Marchese-1983-311-258	258	dormir	1585	SLEEP	to sleep	LES VERBES
+Marchese-1983-311-259	259	enfiler			to thread	LES VERBES
+Marchese-1983-311-260	260	enfler	1573	SWELL	to swell	LES VERBES
+Marchese-1983-311-261	261	entendre	1408	HEAR	to hear	LES VERBES
+Marchese-1983-311-262	262	entrer	749	ENTER	to enter	LES VERBES
+Marchese-1983-311-263	263	faire	2575	DO OR MAKE	to make, to do	LES VERBES
+Marchese-1983-311-264	264	fatiguer	1996	GET TIRED	to get tired	LES VERBES
+Marchese-1983-311-265	265	fermer	1046	SHUT	to close	LES VERBES
+Marchese-1983-311-266	266	finir	1766	FINISH	to finish	LES VERBES
+Marchese-1983-311-267	267	glisser	1324	SLIP	to slip	LES VERBES
+Marchese-1983-311-268	268	gratter			to scratch, to scrape	LES VERBES
+Marchese-1983-311-269	269	insulter	3204	INSULT	to insult	LES VERBES
+Marchese-1983-311-270	270	jurer	1939	SWEAR	to swear	LES VERBES
+Marchese-1983-311-271	271	lancer	1456	THROW	to throw	LES VERBES
+Marchese-1983-311-272	272	laver	1453	WASH	to wash	LES VERBES
+Marchese-1983-311-273	273	(se) laver	3371	WASH (ONESELF)	to wash (oneself)	LES VERBES
+Marchese-1983-311-274	274	macher	321	CHEW	to chew	LES VERBES
+Marchese-1983-311-275	275	manger	1336	EAT	to eat	LES VERBES
+Marchese-1983-311-276	276	marcher	1443	WALK	to walk	LES VERBES
+Marchese-1983-311-277	277	monter	705	GO UP (ASCEND)	to ascend	LES VERBES
+Marchese-1983-311-278	278	mordre	1403	BITE	to bite	LES VERBES
+Marchese-1983-311-279	279	mourir	1494	DIE	to die	LES VERBES
+Marchese-1983-311-280	280	oublier	1523	FORGET	to forget	LES VERBES
+Marchese-1983-311-281	281	pardonner	1528	FORGIVE	to forgive	LES VERBES
+Marchese-1983-311-282	282	penser	1415	THINK (REFLECT)	to think	LES VERBES
+Marchese-1983-311-283	283	percer	1501	BORE	to drill	LES VERBES
+Marchese-1983-311-284	284	prendre	1749	TAKE	to take	LES VERBES
+Marchese-1983-311-285	285	piler	1033	GRIND	to crush, to grind	LES VERBES
```

Comment:
We have french écraser, broyer in Key-2016 mapped to GRIND. Maybe piler tends more towards 1520	CRUSH ?

Reply:
in German, it is rather [zerstampfen](https://de.bab.la/woerterbuch/franzoesisch-deutsch/piler), but this has the meaning of "broyer".

Reply:
Should be then left as it is? German zerstampfen is more like 'to pound', right?

Reply:
German "zerstampfen" is what you do if you want to get some things small, e.g., in cooking. So it is like grinding, but I htink the result is not flower-small.